src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,104 @@
+nav {
+  background: var(--clr-white);
+  box-shadow: var(--light-shadow);
+}
+
+.nav__center {
+  &__header{
```

Comment:
`&__header{` please be consistent with space between _class name_ and _brace_ `&__header {`
Below you have the same issues fix them please.